airbyte-integrations/connectors/source-freshdesk/source_freshdesk/api.py

```diff
@@ -255,6 +258,13 @@ def list(self, fields: Sequence[str] = None) -> Iterator[dict]:
 class TicketsAPI(IncrementalStreamAPI):
     call_credit = 3  # each include consumes 2 additional credits
 
+    def _state_params(self) -> Mapping[str, Any]:
```

Comment:
is this applicable only for tickets? why not apply to all incremental streams? (just asking, not sure what the APi supports)

Reply:
it is a good question, I can think about 
- users that want to sync only specific tickets (because of the size of stream?? mb) 
- and users that want to sync everything after a specific date

so both usages are possible, the general `start_date` will satisfy the second scenario.
To achieve also first one user will need to sync all streams except tickets in `full_refresh` mode and then enable incremental.
So, yes, we can probably introduce `start_date` for all incremental streams